bindings/pyroot_experimental/PyROOT/src/TTreePyz.cxx

```diff
@@ -236,3 +236,155 @@ PyObject *PyROOT::SetBranchAddressPyz(PyObject * /* self */, PyObject *args)
    // Not the overload we wanted to pythonize, return None
    Py_RETURN_NONE;
 }
+
+// Try ( const char*, void*, const char*, Int_t = 32000 )
```

Comment:
Perhaps for our future selves we can comment a bit better the function with doxy?

Reply:
Done!